Website-and-Backend/src/main/resources/templates/index.html

```diff
@@ -94,12 +94,13 @@ <h2><span class="colorLGreen">How to use</span> <span class="colorCream">Ethicli
             <div class="threes">
                 <article class="gallery halfWidth">
                     <img src="images/extension-error.png" alt="Full Extension">
-                    <img src="images/extension-bluesign.png" alt="Full Extension">
+                    <img src="images/extension.png" alt="Full Extension">
                 </article>
                 <article class="imageDesc">
                     <h3>How it Works</h3>
                     <p>While shopping, click on the Ethicli icon in your browser menu to see a breakdown of a shop’s score.
                         Want to know more? Select “View Details” to access links, sources, and suggestions for better products!</p>
+                    <p>Sometimes while shopping, we'll recomend more ethical products based on items that you are viewing.</p>
```

Comment:
```suggestion
                    <p>Sometimes while shopping, we'll recommend more ethical products based on what you're shopping for.</p>
```
recommend has a typo
"items that you are viewing" doesn't sound natural